src/styles/bloks/catalog.scss

```diff
@@ -0,0 +1,123 @@
+.main {
+  &__catalog {
+    margin: 0 auto;
+  }
+}
+
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: catalogWidth($card-width, $card-margin-right);
+  padding: 50px 0;
+
+  &__card {
+    margin-right: $card-margin-right;
+
+    &:nth-last-child(-n + 4) {
+      margin-top: 46px;
+    }
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+  }
+}
+
+.card {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) the 9th point